src/main/java/step2/domain/Lotto.java

```diff
@@ -0,0 +1,54 @@
+package step2.domain;
+
+import step2.exception.LottoException;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Lotto {
+
+    private static final int NUMBER_OF_LOTTONUMBERS = 6;
```

Comment:
띄어쓰기를 추가하면 조금이나마 가독성이 괜찮아질 것 같습니다.
```suggestion
    private static final int NUMBER_OF_LOTTO_NUMBERS = 6;
```